website/docs/examples/file_converter/README.md

```diff
@@ -3,11 +3,31 @@ id: file-converter
 title: File Converter
 ---
 
-
 # File Converter Example
 
-> [This example](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/py/examples/file_converter/main.py)
-will take the `*.bin` log file for a given Oak Camera Stream or snapshot
+:::info Basic Knowledge Requirements
+
+Before diving into this example, here's a quick overview of what you'll need to be familiar with:
+
+1. **Python Programming**: This example is based in Python, making use of various foundational
+constructs like functions, conditional statements, and object-oriented techniques.
+
+2. **File Handling**: The central operation in this example is reading from a `.bin` file and writing
+data to either `.mp4` video files or `.jpg` image files.
+Therefore, understanding how file I/O works in Python, especially with binary files, is crucial.
+
+3. **OpenCV**: The OpenCV library is leveraged extensively in this example, especially for operations
+related to image decoding, processing, and video writing.
+A basic understanding of these aspects of OpenCV is necessary.
```

Comment:
This is pretty extreme for the very basic example at hand... It pretty much just decodes the image, shows it, and saves it as mp4 or jpg